drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_mgmt.c

```diff
@@ -577,6 +581,11 @@ int wifi_nrf_set_twt(const struct device *dev,
 		status = wifi_nrf_fmac_twt_setup(rpu_ctx_zep->rpu_ctx,
 					   vif_ctx_zep->vif_idx,
 					   &twt_info);
+
+		if (status == WIFI_NRF_STATUS_SUCCESS) {
+			vif_ctx_zep->twt_in_progress = true;
```

Comment:
what if the TWT setup is rejected? Will the application teardown explicitly?

Reply:
If TWT setup is rejected there will be no active session, 
application need not explicitly tear down the session.
'twt_in_progress' will be set to false, if for some reason twt setup failed/rejected.

Reply:
I think, I mentioned ''need not" tear down the session explicitly

Reply:
Ah sorry missed the *not*, and we are setting to false in the event, thanks.